components/integration/base/delete-snapshots.yaml

```
@@ -0,0 +1,25 @@
+kind: ClusterRole
+apiVersion: authorization.openshift.io/v1
+metadata:
+  name: delete-snapshots
+rules:
+  - apiGroups:
+    - "appstudio.redhat.com"
+    resources:
+      - "snapshots"
+    verbs:
+      - "delete"
+      - "deletecollection"
+---
+kind: ClusterRoleBinding
+apiVersion: authorization.openshift.io/v1
```

Same reason as above
```suggestion
rbac.authorization.k8s.io/v1
```